### DISCUSSION TITLE
Approval for WalletConnect explorer: Gryfyn Wallet

### DISCUSSION THREAD

**Discussion by arnoldshum-hextrust** [WalletConnect/walletconnect-monorepo#2023] [Developer Support] [answered]
Hi WalletConnect Team, 

I was wondering if there's any feedback on our application for listing our Gryfyn wallet onto the WalletConnect Explorer? Thanks!

Project ID: e5ecd4ac096c3c131d87095e72614230

**Comment by atifnimran** [comment]
Sir help

**Comment by bkrem** [comment, marked as answer]
Approved! ✅

**Reply by arnoldshum-hextrust** [reply]
thank you!

**Reply by atifnimran** [reply]
Tnx
